cluster/stone-prd-rh01/tenants/trusted-content-tenant/apps.yaml

```diff
@@ -23,3 +23,39 @@ spec:
       url: https://github.com/trustification/guac
       dockerfileUrl: dockerfiles/Dockerfile.guac-ubi
       context: ./
+---
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Component
+metadata:
+  name: trustification-service
+  annotations:
+    build.appstudio.openshift.io/request: configure-pac
+    image.redhat.com/generate: '{"visibility": "private"}'
+spec:
+  application: trustification
+  componentName: trustification-service
+  source:
+    git:
+      revision: stable
+      devfileUrl: https://raw.githubusercontent.com/devfile-samples/devfile-sample-go-basic/main/devfile.yaml
```

Comment:
I don't think this is needed

Reply:
@ernesgonzalez33 what we saw yesterday is that without this field Argo fail to apply the change since the `source` section of the component can't be updated.

Reply:
Oh! Ok. I'll resolve, approve and merge then